### DISCUSSION TITLE
Shiny App Error

### DISCUSSION THREAD

**Discussion by rafiff23** [teamalgoritma/community#597] [Q&A] [answered]
Setiap saya run shiny pada pertama kali semua nya lancar tanpa ada kendala tapi saat ingin dijalankan kedua kalinya di terminal hanya muncul runApp() dan harus di close dulu R nya dibuka kembali baru bisa lagi

**Comment by fafilia** [comment, marked as answer]
Dear @rafiff23 ,
Terima kasih untuk pertanyaan yang diajukan. Hal tersebut bukanlah sebuah kendala karena memang ketika kita ingin menjalankan ulang aplikasi Anda bisa menutup terlebih dahulu aplikasi yang sedang berjalan, namun pada keadaan ini Bapak tidak perlu untuk menutup R/Rstudio melainkan hanya menutup jendela baru yang biasanya muncul saat kita running aplikasi dashboardnya. Hal lain yang bisa Anda lakukan jika tidak ingin menutup jendela baru tersebut, Anda bisa klik tombol Reload App yang terdapat di bagian atas sama halnya saat Anda klik Run App. Demikian infromasi yang dapat saya sampaikan, semoga cukup menjelaskan. Terima kasih.
